design-doctorine.md

```diff
@@ -31,7 +31,11 @@
 
 ## Clinical data points
 * this data will already be available in the health information system (HIT) as part of patient care
-* aim is to extract it for purposes of analytics
+  * aim is to extract it for purposes of analytics
+* will be aligning codes, valuesets, and extensions used with US Core and mCODE
+  * but not introducing a hard dependency on those guides or deriving profiles from them since we are Universal Realm and those are US specs
+* will not be fixing minimum cardinality of elements since this is a low-level, domain of knowledge specification
+  * more concrete implementation guides will be needed here for actual implementation that fix them
 
```

Comment:
```suggestion
* the binding of the valuesets will be set to required to represent the ICHOM BC set
* in case of drug treatments, such as hormonal and immunotherapy, we will cover all the different types of medication resources and have implementers decide for themselves what fits their needs.

```